doc/polycli_loadtest.md

```diff
@@ -114,7 +114,7 @@ The codebase has a contract that used for load testing. It's written in Yul and
       --chain-id uint                           The chain id for the transactions.
   -c, --concurrency int                         Number of requests to perform concurrently. Default is one request at a time. (default 1)
       --contract-address --mode contract-call   The address of the contract that will be used in --mode contract-call. This must be paired up with `--mode contract-call` and `--calldata`
-      --contract-call-payable --function-sig    Use this flag if the --function-sig is a `payable` function, the value amount passed will be from `--eth-amount`. This must be paired up with `--mode contract-call` and `--contract-address`
+      --contract-call-payable --eth-amount      Use this flag if the function is payable, the value amount passed will be from --eth-amount. This must be paired up with `--mode contract-call` and `--contract-address`
```

Comment:
I didn't notice that before but it looks like we may have to clean up the flags description. I'll create a ticket for that.